Mathlib/Data/Finset/Image.lean

```diff
@@ -79,6 +79,13 @@ theorem mem_map_equiv {f : α ≃ β} {b : β} : b ∈ s.map f.toEmbedding ↔ f
       simpa, fun h => ⟨_, h, by simp⟩⟩
 #align finset.mem_map_equiv Finset.mem_map_equiv
 
+theorem subset_map_equiv_iff {t : Finset β} {f : α ≃ β} : t ⊆ s.map f ↔ t.map f.symm ⊆ s := by
```

Comment:
Can you make sure this matches a `Set` lemma with a similar name?

Reply:
🏓

Reply:
Done? (`Finset.map_subset_iff`)